medusa/providers/torrent/json/t411.py

```diff
@@ -82,7 +82,24 @@ def __init__(self):
 
         # Miscellaneous Options
         self.headers.update({'User-Agent': USER_AGENT})
-        self.subcategories = [433, 637, 455, 639]
+        self.subcategories = [433, 637, 455, 639, 636]
```

Comment:
This would be better documented like this:
```python
self.subcategories = [
    433,  # Série TV
    455,  # Animation
    ...
    # 402,  # Vidéo-clips
]
```